### ISSUE TITLE
Improve the UI for displaying the Wikipedia description

### ISSUE THREAD

**Issue by RuslanFatikhov** [organicmaps/organicmaps#9943]
⚠ Have you searched for similar, already existing issues?  
Yes.

**Is your feature request related to a problem? Please describe.**  
When opening a place description in full-screen mode in Organic Maps, the text formatting lacks proper margins, making it appear cramped and less readable. This issue reduces the visual appeal and readability of the description text, as shown in the attached screenshot.

**Describe the ideal solution**  
To improve readability and design, add padding of **16px** on the sides and **24px** on the top and bottom of the text block. This adjustment will create a cleaner and more user-friendly layout. I’ve attached an example screenshot demonstrating the ideal layout with the proposed padding.

**Describe alternatives you have considered**  
Currently, there are no alternatives within the app, and users have to deal with the cramped layout. However, other apps like 2GIS implement a better design for similar text blocks by using proper padding and spacing in their description cards. Attached is a screenshot from 2GIS that demonstrates a more user-friendly approach to displaying such information.

**Additional context**  
Attached are screenshots showing:  
1. The current issue with the lack of padding.  
2. An example of the proposed solution with proper padding applied.  
3. A reference from 2GIS, showcasing an optimized layout for similar descriptions.

---
Problem
![IMG_4781](https://github.com/user-attachments/assets/e8caa318-1437-47f7-b661-60d03818a5fa)

Reference
![IMG_4782](https://github.com/user-attachments/assets/82db75a6-ffb6-4c7d-9166-45aa4eb24cf6)

Solution
![solution](https://github.com/user-attachments/assets/26cabe22-8bbe-4ddd-9e62-c2c581431890)

**Comment by RedAuburn** [comment]
Agreed, padding looks much better! Although it may be better to use percentage sizes to scale with different screen sizes better.

here's the webcontainer fragment: https://github.com/organicmaps/organicmaps/blob/master/android/app/src/main/java/app/organicmaps/widget/placepage/PlaceDescriptionFragment.java

**Comment by DeshDeepakKant** [comment]
@RedAuburn  I couldn't locate the .xml layout file for this page. It seems that the description displayed here is dynamically fetched from Wikimedia and rendered directly in the app.

**Comment by RedAuburn** [comment]
> @RedAuburn I couldn't locate the .xml layout file for this page. It seems that the description displayed here is dynamically fetched from Wikimedia and rendered directly in the app.

yep, it's a webview

**Comment by AffanShaikhsurab** [comment]
looks like an intersting issue , can someone assign me please , i would love to work on it !!

**Comment by DeshDeepakKant** [comment]
> looks like an intersting issue , can someone assign me please , i would love to work on it !!

I am already working  on it

**Comment by RuslanFatikhov** [comment]
Nice! Thank you!
